awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx

```diff
@@ -62,6 +61,38 @@ function JobTemplateForm({
     Boolean(template?.host_config_key)
   );
 
+  const formikBag = useFormikContext();
```

Comment:
If you do the other things mentioned, you won't need to call useFormikContext or have a hold of the formikBag